plan.sh

```diff
@@ -17,11 +18,6 @@ pkg_deps=(
 )
 pkg_bin_dirs=(bin)
 
-pkg_version() {
-  export LC_ALL=en_US LANG=en_US
-  pip search --disable-pip-version-check awscli | grep '^awscli (' | awk -F'[()]' '{print $2}'
-}
-
 do_before() {
```

Comment:
we should remove this `do_before` as well

Reply:
👍